go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -273,25 +226,11 @@ func (ovn *Controller) deleteLoadBalancerRejectACL(lb, vip string) {
 		return
 	}
 
-	switches, err := ovn.getLogicalSwitchesForLoadBalancer(lb)
+	_, stderr, err := util.RunOVNNbctl("remove", "port_group", ovn.clusterPortGroupUUID, "acls", acl)
 	if err != nil {
-		klog.Errorf("Could not retrieve logical switches associated with load balancer %s", lb)
+		klog.Errorf("Failed to remove reject ACL for %s from LB %s: stderr: %q, error: %v", vip, lb, stderr, err)
 		return
 	}
 
-	args := []string{}
```

Comment:
We presumably need to handle the upgrade case, where a LB is created in the old style and deleted in the new style.

Reply:
Yeah I'll work on this more next week after PTO.

Reply:
@dcbw 
Seems like the name we use for LB ACL is lb-<source-ip>-<source-port>. Given that these ACL's will now move to port-group, if we find any existing ACL with name lb-*-*  in the ACL table, we can remove that reference from the "acls" column of the Logical_Switch table. And if the removed ACL is not referenced by any other table (e.g Port Group) it will automatically be removed from the ACL table because of "isRoot:false". This solution might work, but at scale, it can be compute intensive during upgrade, because it will try to find individual lb-*-* entries in each Logical_switch and remove it, so it can take a while at higher scale.

Just throwing another thought, that might work much better. We should create a port-group for each Logical_Switch and move all the ACLs (lb,egressfirewall,policy) to that port-group. That way, during the upgrade, once we process all ACLs' we can just go and wipe the "acls" column for the Logical_Switch. That would be a pretty quick transaction. Looking at the egressFirewall and Policy, it seems like Policy is already using a port group for most of its ACL. That leaves egressFirewall, which doesn't seem like a very huge undertaking (i might be wrong ). 

Obviously this approach won't be as efficient as adding all the logical ports to one port group and applying the ACL ( 1 ACL for all the nodes, with an enormous port group (logical_switch * logical*port). But in this case it will apply that ACL on each port group, so for 100 nodes, it's uuid will be added to each node's port group, so total 100 * 48 bytes, compared to the current implementation, where we add 1 ACL per service and uuid to acls column of Logical_Switch (services * 48 * 100). 
But the above approach will give us better structure to avoid similar kinds of issues in future, and better approach to handle the ACLs.

Reply:
@girishmg @abhat @danwinship @squeed any thoughts on above proposal?

Reply:
@dcbw port_group per node, won't work because we still will have to create deny acl per service. Applying them per node port-group won't help in reducing the overall number of entries in acl column. I went with the same approach of using single port group across all nodes and added the cleanup logic. 

I tried to push the patch to this PR, but it was denied. So i pushed a new PR #1711